sdk/src/main/java/com/qonversion/android/sdk/public/PeriodType.kt

```diff
@@ -0,0 +1,3 @@
+package com.qonversion.android.sdk.public
```

Comment:
I guess all the dto objects should be placed in the corresponding package inside `public`